users/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+class User(models.Model):
+    email           = models.EmailField(max_length=254)
+    password        = models.CharField(max_length=300)
+    author_name     = models.CharField(max_length=30, unique=True)
+    author_job      = models.CharField(max_length=30, blank=True)
+    author_intro    = models.TextField(blank=True)
+    create_time     = models.DateTimeField(auto_now_add=True)
+    update_time     = models.DateTimeField(auto_now=True)
+    post            = models.ManyToManyField('posts.Post', through='users.Like')
```

Comment:
likes로 수정을 해야될 것 같습니다.

이유는 User가 작성한 post 목록도 있을 것이고 좋아요한 post 목록도 있을 것입니다.
그렇기 때문에 user가 좋아요한 게시글을 불러올 때에는 user.post가 아니라 `user.likes` 또는 `user.liked_posts` 코드로 데이터를 가져올 수 있도록 컬럼명을 수정해주셔야 합니다~

```suggestion
    likes = models.ManyToManyField('posts.Post', through='users.Like')
```